pkg/security/ebpf/c/defs.h

```diff
@@ -227,14 +227,17 @@ enum event_type
     EVENT_ALL = 0xffffffff // used as a mask for all the events
 };
 
+enum {
+    EVENT_FLAGS_ASYNC = 1, // async, mostly io_uring
```

Comment:
2 questions:
- could we set async to `1 << 0` ?
- why is save by ad set to `1 << 2` and not `1 << 1` ? Aren't we missing one bit here ?